File: MicroserviceApp/Backend/AdvertManager/src/main/java/com/example/advertmanagerapp/model/Capture.java
package com.example.advertmanagerapp.model;

import lombok.Data;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
@Table
public class Capture {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date takeDate;

    @Temporal(TemporalType.TIMESTAMP)
    @Column
    private Date leaveDate;

}

File: MicroserviceApp/Frontend/microservice-ng-app/src/app/model/capture.ts
export class Capture {
  id: number;
  takeDate: Date;
  leaveDate: Date;
  advertId: number;

  constructor(id: number, take: Date, leave: Date,adv:number) {
    this.id=id;
    this.takeDate=take;
    this.leaveDate=leave;
    this.advertId=adv
  }

}
